### ISSUE TITLE
hung on restart - either shutting down or restarting

### ISSUE THREAD

**Issue by gunterzielke** [SiCKRAGETV/sickrage-issues#982]
The problem usually starts out with a situation where I click on a show to open it - and it does not, it sits there waiting and nothing does. I can still select some other pages in the system but can not open any shows.
After that the next action is to try to restart sick beard - sometimes (but rarely) it works, but mostly the restart hangs either in shutting down, or restarting.
When the hung-up happens in shutting down, I sometimes can save it by "kill -9 pid" from the command line but often the restarting just hangs.
When the hung up happens on the start-up, so far the only way to get it going again is a reboot of the whole machine (an Ubuntu, by the way).
I would really appreciate any help you might be able to give me here.
Thanks, Gunter

**Comment by fernandog** [comment]
Disable:
auto update on startup
Backlog search on startup

**Comment by gunterzielke** [comment]
@fernandog 
thank you for the answer - it did  get better but today I had another hung again.
Here some more info that might help to track that down.
I went to Restart in Sick Beard and it hung on 'Shutting down' I could see that it was trying to close process 1828 and so, from the command line, issued a kill 1828 - which did  nothing, but a 
kill -9 1829 caused to return a 'Success' for the shutdown part in the interface in Sick Beard. But it then hung in the starting and I had to reboot the computer to get out of that.
Thanks, Gunter
